packages/features/settings/appDir/SettingsLayoutAppDirClient.tsx

```diff
@@ -672,6 +672,7 @@ export type SettingsLayoutProps = {
   children: React.ReactNode;
   currentOrg: Awaited<ReturnType<typeof OrganizationRepository.findCurrentOrg>> | null;
   otherTeams: Awaited<ReturnType<typeof OrganizationRepository.findTeamsInOrgIamNotPartOf>> | null;
+  containerClassName?: string;
```

Comment:
Effort to be up-to-date with `SettingsLayout.tsx` (used for pages router layout)